src/main/java/com/minecolonies/coremod/items/ItemScanTool.java

```diff
@@ -89,16 +99,16 @@ else if (!compound.hasKey("pos2"))
         }
         else
         {
-            @NotNull final BlockPos pos1 = BlockPosUtil.readFromNBT(compound, "pos1");
-            @NotNull final BlockPos pos2 = BlockPosUtil.readFromNBT(compound, "pos2");
+            @NotNull final BlockPos pos1 = BlockPosUtil.readFromNBT(compound, FIRST_POS_STRING);
+            @NotNull final BlockPos pos2 = BlockPosUtil.readFromNBT(compound, SECOND_POS_STRING);
 
             //todo if on client (ssp) -> no need to send message, we can execute it in worldIn.isRemote without a message.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)